cmd/deploy/deploy.go

```diff
@@ -450,7 +448,6 @@ func (dc *DeployCommand) deploy(ctx context.Context, opts *Options) error {
 		oktetoLog.Information("Running '%s'", command.Name)
 		oktetoLog.SetStage(command.Name)
 		if err := dc.Executor.Execute(command, opts.Variables); err != nil {
-			oktetoLog.AddToBuffer(oktetoLog.ErrorLevel, "error executing command '%s': %s", command.Name, err.Error())
```

Comment:
This was being logged along with the one in main.go, so in this case we just return error and the one handling this error (which is main.go) should be the one responsible of logging the message